### PR TITLE
feat(hyperlink-icon): add hyperlink icon

### PR DESCRIPTION
**Pull request by shamshad17**
# Icon added
Screenshot 2022-03-11 at 10 40 46 AM

### CODE REVIEW

**Review by shridhar52war — APPROVED** [review]

File: packages/blade-old/src/icons/HyperLink/HyperLink.native.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { Path } from 'react-native-svg';
+import Icon, { IconPropTypes } from '../../atoms/Icon';
+
+function HyperLink(props) {
+  return (
+    <Icon viewBox="0 0 20 20" {...props}>
```
Comment (shubhnik) [inline packages/blade-old/src/icons/HyperLink/HyperLink.native.js:7]:
```suggestion
    <Icon viewBox="0 0 24 24" {...props}>
```
@shamshad17 this should be the viewbox like in other icons, don't edit it to 24 directly here. It should be imported this way from the icon. Can you point us to the icon where you are importing it from? // @shridhar52war

**Review by shubhnik — CHANGES_REQUESTED** [review]

File: packages/blade-old/src/icons/HyperLink/HyperLink.native.js (hunk 1 above)
Comment (shamshad17) [reply packages/blade-old/src/icons/HyperLink/HyperLink.native.js:7]: [Figma](https://www.figma.com/file/4GTOYOoG28EwW3mFXJh1Ik/Mobile-app-Navigation?node-id=2206%3A110788)

If I change viewbox to 24 icon becomes smaller in comparison to other icons. @shubhnik

Comment (shubhnik) [reply packages/blade-old/src/icons/HyperLink/HyperLink.native.js:7]: @shamshad17 we don't explicitly edit this property, it is exported from Figma directly. // @shridhar52war

Comment (shridhar52war) [reply packages/blade-old/src/icons/HyperLink/HyperLink.native.js:7]: yes, @shubhnik rightly mentioned. @shamshad17 While exporting the svg, expose it with view box of "24 24".

**Comment by gupta-ji6** [comment]
@shamshad17 add a changeset with `minor` bump in `@razorpay/blade-old` package

File: packages/blade-old/src/icons/HyperLink/HyperLink.native.js (hunk 1 above)
Comment (shamshad17) [reply packages/blade-old/src/icons/HyperLink/HyperLink.native.js:7]: Got it. I'll ask designer to update the svg.

Comment (shubhnik) [reply packages/blade-old/src/icons/HyperLink/HyperLink.native.js:7]: @shamshad17 btw we already have a link icon, looks like it is similar and should suffice? https://github.com/razorpay/blade/tree/master/packages/blade-old/src/icons/Link @gupta-ji6

Comment (shamshad17) [reply packages/blade-old/src/icons/HyperLink/HyperLink.native.js:7]: @shubhnik Currently we have this Link icon.
Screenshot 2022-03-11 at 3 10 11 PM

**Review by shridhar52war — APPROVED** [review]

**Review by shubhnik — APPROVED** [review]

**Pull request merged**